src/main/webapp/app/therapeuticareas/views/therapeutic.area.questionnaire.view.htm

```diff
@@ -103,119 +37,57 @@
 
                 <div class="question">
                     <div class="input-group">
-                        <input type="text" class="form-control" data-ng-model="newAnswer" placeholder="Enter a new answer..."/>
-                          <span class="input-group-btn">
-                            <button class="btn btn-default" type="button" data-ng-click="addAnswer(questionnaireEntry, newAnswer)"
-                                    data-ng-disabled="!newAnswer" title="Add">
-                                <span class="glyphicon glyphicon-plus"></span>
-                            </button>
-                          </span>
+                        <input id="inp_newAnswer" type="text" class="form-control" data-ng-model="newAnswer"
```

Comment:
Suffix the newAnswer button with the questions sortOrder as there's one newAnswer button per question
<input ng-attr-id="{{ 'inp_newAnswer_' + questionnaireEntry.question.sortOrder }}"